src/components/TheTransactionList.vue

```diff
@@ -19,14 +25,13 @@
           <button
             class="delete"
             aria-label="close"
-            @click="
-              ;(showModal = false),
-                (showCreditRefreshNotification = false),
-                $store.commit('setModalState', false)
-            "
+            @click="closeTransactionListModal()"
```

Comment:
here you use a function to do 2 instructions, which is fine, but then why not do that on all other modals ? or use the two instructions here.